src/main/java/hec/ensemble/EnsembleConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package hec.ensemble;
+import java.time.Duration;
+import java.time.ZonedDateTime;
+import hec.stats.Configuration;
+
+
+public class EnsembleConfiguration implements Configuration{
```

Comment:
EnsembleConfiguration is very close to Ensemble, can they be merged?

Reply:
they are, and i think they can, can we have Ensemble be composed of EnsembleConfiguration and the values[][] and the parent ensembleTS?

Reply:
Ok.  So EnsembleConfiguration (EnsembleDefinition/EnsembleSelection) can be  passed as an arg to optional Ensemble Constructor?

Reply:
could be that way, or the current constructor could be used to construct a EnsembleConfiguration and store it instead of the three parameters it incapsulates.